files/en-us/web/api/channel_messaging_api/index.html

```diff
@@ -18,7 +18,7 @@ <h2 id="Channel_messaging_concepts_and_usage">Channel messaging concepts and usa
 
 <p>A message channel is created using the {{domxref("MessageChannel.MessageChannel", "MessageChannel()")}} constructor. Once created, the two ports of the channel can be accessed through the {{domxref("MessageChannel.port1")}} and {{domxref("MessageChannel.port2")}} properties (which both return {{domxref("MessagePort")}} objects.) The app that created the channel uses <code>port1</code>, and the app at the other end of the port uses <code>port2</code> — you send a message to <code>port2</code>, and transfer the port over to the other browsing context using {{domxref("window.postMessage")}} along with two arguments (the message to send, and the object to transfer ownership of, in this case the port itself.)</p>
 
-<p>When these transferable objects are transferred, they are 'neutered' on the previous context — the one they previously belonged to. For instance a port, when is sent, cannot be used anymore by the original context.</p>
+<p>When these transferable objects are transferred, they are deactivated on the context they previously belonged to. A port, after it is sent, can no longer be used by the original context.</p>
```

Comment:
```suggestion
<p>When these transferable objects are transferred, they are no longer usable on the context they previously belonged to. A port, after it is sent, can no longer be used by the original context.</p>
```
*“no longer usable”* is the term that the spec itself has at https://html.spec.whatwg.org/multipage/web-messaging.html#dom-messageport-postmessage-options-dev